deploy/backplane/osdfm/20-osdfm.SubjectPermission.yml

```
@@ -0,0 +1,14 @@
+apiVersion: managed.openshift.io/v1alpha1
+kind: SubjectPermission
+metadata:
+  name: backplane-osdfm
+  namespace: openshift-rbac-permissions
+spec:
+  clusterPermissions:
+  - backplane-readers-cluster
+  permissions:
+  - clusterRoleName: dedicated-readers
+    namespacesAllowedRegex: "(^kube$|^kube-.*|^openshift$|^openshift-.*|^default$|^redhat-.*)"
```

```suggestion
    namespacesAllowedRegex: "(^kube-.*|^openshift$|^openshift-.*|^default$|^redhat-.*)"
```

`kube` NS doesn't actually exist, guessing that was carried from another place.

---

Will make this change.

As for the card, its not closed but the parent was: https://issues.redhat.com/browse/SDA-8133 I re-opened.